mockServer/data/flowers/flowersData.ts

```diff
@@ -0,0 +1,191 @@
+export const flowersData = [
+  {
+    _id: "7128d52b4bd76bf2e3a75519",
+    name: "Flowering dogwood",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1631103160/icons/flowering_dogwood_wnbw7d.gif",
+    price: 15,
+    isActive: true,
+    isSecret: false,
+  },
+  {
+    _id: "7128d52b4bd76bf2e3a755a4",
+    name: "Marigold",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1631103161/icons/marigold_timpbu.gif",
+    price: 4,
+    isActive: true,
+    isSecret: false,
+  },
+  {
+    _id: "7128d52b4bd76bf2e3a755a1",
+    name: "Dahlia",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1631103160/icons/dahlia_jemyso.gif",
+    price: 7,
+    isActive: true,
+    isSecret: false,
+  },
+  {
+    _id: "7128d52b4bd76bf2e3a75518",
+    name: "Chamomile",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1631103160/icons/cherry_blossom_fvhcq5.gif",
+    price: 10,
+    isActive: true,
+    isSecret: false,
+  },
+  {
+    _id: "7128d52b4bd76bf2e3a75516",
+    name: "Cosmos",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1631103160/icons/cosmos_isdopf.gif",
+    price: 4,
+    isActive: true,
+    isSecret: false,
+  },
+  {
+    _id: "7128d52b4bd76bf2e3a75515",
+    name: "Begonia",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1631103159/icons/begonia_nz6tuy.gif",
+    price: 8,
+    isActive: true,
+    isSecret: false,
+  },
+  {
+    _id: "7128d52b4bd76bf2e3a75511",
+    name: "Iris",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1630492453/icons/iris_przxuc.gif",
+    price: 5,
+    isActive: true,
+    isSecret: false,
+  },
+  {
+    _id: "7128d52b4bd76bf2e3a75517",
+    name: "Cherry blossom",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1631103160/icons/cherry_blossom_fvhcq5.gif",
+    price: 20,
+    isActive: true,
+    isSecret: false,
+  },
+  {
+    _id: "8128d52b4bd76bf2e3a755a0",
+    name: "sunflower",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1631103161/icons/sunflower_eaf66x.gif",
+    price: 5,
+    isActive: true,
+    isSecret: false,
+  },
+  {
+    _id: "7128d52b4bd76bf2e3a75513",
+    name: "Balloon flower",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1631103159/icons/balloon_flower_ut70yg.gif",
+    price: 2,
+    isActive: true,
+    isSecret: false,
+  },
+  {
+    _id: "612789b765c2e6a7e7e76bd3",
+    name: "Anemone",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1630492426/icons/anemone_egth6w.gif",
+    price: 3,
+    isActive: true,
+    isSecret: false,
+  },
+  {
+    _id: "7128d52b4bd76bf2e3a755a5",
+    name: "Lily",
+    imageURL:
+      "https://res.cloudinary.com/dyvrke6ml/image/upload/v1631103160/icons/lily_s6luhq.gif",
+    price: 8,
+    isActive: true,
```

Comment:
Added mock data so you can test buying locally without accessing the database, so you have infinite tries and coins👍